Workspace.swift

```diff
@@ -1,6 +1,42 @@
 import WorkspaceConfiguration
 
 let configuration:WorkspaceConfiguration = .init()
+
+// ••••••• General •••••••
+
+// The project appears to only target Linux.
+// In any case, it fails to compile on macOS right now.
+configuration.supportedOperatingSystems.remove(.macOS)
+configuration.supportedOperatingSystems.remove(.iOS)
+configuration.supportedOperatingSystems.remove(.watchOS)
+configuration.supportedOperatingSystems.remove(.tvOS)
+
+// ••••••• Proofreading •••••••
+
+// These are documented at https://sdggiesbrecht.github.io/Workspace/🇨🇦EN/Types/ProofreadingRule.html
+
 configuration.proofreading.rules.remove(.unicode)
 configuration.proofreading.rules.remove(.colonSpacing)
+
+// Disabled because there are violations.
+// For an explanation, see: https://forums.swift.org/t/introducing-documentation-generation-for-swift-packages/15971/8?u=sdggiesbrecht
+configuration.proofreading.rules.remove(.compatibilityCharacters)
+
+// The rules may conflict with the PNG project style:
+// Would require parameter callouts to be grouped.
+configuration.proofreading.rules.remove(.parameterGrouping)
+// Would require repeated line‐style documentation instead of block‐style,
+// which is vulnerable to Xcode’s autoindent destroying semantic Markdown indents.
+configuration.proofreading.rules.remove(.autoindentResilience)
```

Comment:
tbh the only reason I am using `/** */` is because Atom lacks the tooling to triple-slash a paragraph. could Workspace fix these (and possibly do linebreaking) automatically? just a feature request

Reply:
`/** */` is actually more logical. When the day comes where Xcode stops breaking the indentation, the rule may even be reversed. If your contributors largely don’t use Xcode anyway, then I actually recommend keeping the rule disabled and continuing as you do now.

But in answer to your question, there is no automation for that at the moment.

Reply:
Also answered I think.